js/src/collapse.js

```diff
@@ -314,6 +325,31 @@ const Collapse = (($) => {
       }
     }
 
+    _getShowClass() {
+      const tabClass = this._element.classList
+
+      // Detect flex for inline style
+      let useFlex = $(this._element).css('display').indexOf('flex') !== -1
```

Comment:
How about just using https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes?

Reply:
I checked `babel-polyfill` and currently when we import babel-polyfill we will import all the available polyfill not the `includes` one so for now we should merge this PR and open an issue on how to integrate babel-polyfill with only specific es6 polyfill

Reply:
No worries, it's not so important currently as long as we don't use features that aren't available for the browsers we support.